src/uct/ib/base/ib_md.c

```diff
@@ -205,36 +205,61 @@ static ucs_stats_class_t uct_ib_md_stats_class = {
 };
 #endif
 
+/*
+ * - overhead calculations:
+ *   Gen1/2:
+ *     Start   SeqNum   Hdr_64bit   LCRC   End
```

Comment:
what about `ECRC` (4 bytes) for Gen1/2 and Gen3/4?

Reply:
it's optional (usually off)

Reply:
thanks for the exaplanation!